code/modules/mining/lavaland/weather_types.dm

```diff
@@ -65,7 +65,7 @@
 	start_up_time = 300 //30 seconds
 	start_up_message = "The wind begins to pick up. Seek shelter."
 	duration_lower = 60 //1 minute
-	duration_upper = 180 //3 minutes
+	duration_upper = 120 //3 minutes
```

Comment:
Will NOT be merged unless you fix this. Come on dude, get your shit together. You should know better.